charts/umbrella/values.yaml

```diff
@@ -371,115 +298,184 @@ centralidp:
+        - clientId: "sa-cl7-cx-7"
+          clientSecret: "changeme"
+        - clientId: "sa-cl8-cx-1"
+          clientSecret: "changeme"
+        - clientId: "sa-cl21-01"
+          clientSecret: "changeme"
+        - clientId: "sa-cl22-01"
+          clientSecret: "changeme"
+        - clientId: "sa-cl24-01"
+          clientSecret: "changeme"
+        - clientId: "sa-cl25-cx-1"
+          clientSecret: "changeme"
+        - clientId: "sa-cl25-cx-2"
+          clientSecret: "changeme"
+        - clientId: "sa-cl25-cx-3"
+          clientSecret: "changeme"
+    bpn: "BPNL00000003CRHK"
+    sharedidp: "http://sharedidp.tx.test"
+    # -- test service accounts for EDC - MIW which are obsolete since R24.05;
+    # uncomment once EDC uses SSI DIM Wallet Stub and the helm chart testing has been updated;
+    # currently the post-install testdata-upload-job fails if not available
+    extraServiceAccounts:
+      clientSecretsAndBpn:
+        - clientId: satest01
+          clientSecret: UbfW4CR1xH4OskkovqJ2JzcwnQIrG7oj
+          bpn: BPNL00000003AZQP
+        - clientId: satest02
+          clientSecret: pyFUZP2L9UCSVJUScHcN3ZEgy2PGyEpg
+          bpn: BPNL00000003AYRE
+        - clientId: satest03
+          clientSecret: tPwy4exxH1sXBRQouobSA2nNVaaPuwCs
+          bpn: BPNL00000003AVTH
+        - clientId: satest04
+          clientSecret: BxZ3cwYUPJKK7gI4wq7q6Hgoxel6MphF
+          bpn: BPNL00000003AWSS
+        - clientId: satest05
+          clientSecret: dR00GN1AWCYbRGbZY8TXjs2YEPMeCxLF
+          bpn: BPNL00000003B0Q0
+        - clientId: satest06
+          clientSecret: pDSziT0TUFAkMx0qGFcvpE4XkMqPh13v
+          bpn: BPNS0000000008ZZ
+        - clientId: satest07
+          clientSecret: GY5a44sNuNIjrTyjHvdEPLeNRHH0Kt39
+          bpn: BPNL00000003CNKC
+        - clientId: satest08
+          clientSecret: WUXpQx1aIclA7enqtk4o2uvLDLMreUMI
+          bpn: BPNL00000003B6LU
+        - clientId: satest09
+          clientSecret: N08TGNdhUskJcmVEnOh1tAGwr9oca9PU
+          bpn: BPNL00000003CML1
+        - clientId: satest10
+          clientSecret: gzdSG0CBDJrtv1gje0zUASu1S9P4I7xP
+          bpn: BPNS00000008BDFH
+        - clientId: satest11
+          clientSecret: CC3fz3dQGZsBp2NCbowOV65efBFZTgEO
+          bpn: BPNL00000003B2OM
+        - clientId: satest12
+          clientSecret: 2gjSlFxBO7spEM4aTz3f8CqDS0klbt7C
+          bpn: BPNL00000003CSGV
+        - clientId: satest13
+          clientSecret: 3YQzDqEsdUZ83DVHSIRYUCK4pot61r5M
+          bpn: BPNL00000003B5MJ
+        - clientId: satest14
+          clientSecret: 7qtMpfN3otq5dGiEPssVongXK56lb9LE
+          bpn: BPNL00000003AXS3
+        - clientId: satest15
+          clientSecret: 8QiZ8ineW0Lt8ZOlC2MYuCR0TvM6vMYX
+          bpn: BPNL00000003B3NX
+        - clientId: satest16
+          clientSecret: d2sqUurBH9Vd8DNRmjiMfObU67ajorCq
+          bpn: BPNL00000000BJTL
+    initContainer:
+      image:
+        name: docker.io/tractusx/umbrella-init-container:1.1.0-init
```

Comment:
I understand that the **1.1.0-init tag** still needs to be created so that the following GitHub Actions can generate it: [build-init-container](https://github.com/eclipse-tractusx/tractus-x-umbrella/actions/workflows/build-init-container.yml)

Reply:
I just pushed the tag https://github.com/eclipse-tractusx/tractus-x-umbrella/releases/tag/1.1.0-init, so that then new init container got built